payments_service/solitude/tests/test.py

```diff
@@ -69,6 +76,32 @@ def test_pass_post_params_to_solitude(self):
         eq_(call_args[0][0]['foo'], params['foo'])
         eq_(call_args[0][0]['bar'], params['bar'])
 
+    def test_pass_get_params_to_solitude(self):
+        params = {'foo': '1', 'bar': 'baz'}
+        self.get(query_params=params)
+        assert self.solitude.services.status.get.called
```

Comment:
I think you can remove this line because if it didn't get called the next assertion would fail.